### ISSUE TITLE
How you update?

### ISSUE THREAD

**Issue by alexeydg** [anouarabdsslm/laravel-paypalpayment#163]
`Updated on 14/5/2020: I will update this package by the end of this week`

**Comment by anouarabdsslm** [comment]
yes I found out that paypal SDK is deprecated https://github.com/paypal/PayPal-PHP-SDK I will try to check there new packages to see how I can re-implement it with the package
